scripts/2009/01/sitemaps/sitemap.py

```diff
@@ -45,38 +45,50 @@ def xopen(filename):
     else:
         return open(filename)
 
+def urlsafe(name):
+    """Slugifies the name to produce OL url slugs
+
+    XXX This is duplicated from openlibrary.core.helpers because there
+    isn't a great way to import the methods from openlibrary as a
+    package
+    """
+    # unsafe chars according to RFC 2396
+    reserved = ";/?:@&=+$,"
+    delims = '<>#%"'
+    unwise = "{}|\\^[]`"
+    space = ' \n\r'
```

Comment:
These are good points -- this code is based on https://github.com/internetarchive/openlibrary/blob/master/openlibrary/core/helpers.py's _get_safepath_re -- I'm hesitant to make changes which will make the two behave differently. Let's merge this branch and I'll fork this as an issue to attack the problem at the source (helpers.py)